src/KeyVault/KeyVault/Commands/CamelCasePropertyNamesWithOverridesContractResolver.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Newtonsoft.Json.Serialization;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+using System.Linq;
+
+namespace Microsoft.Azure.Commands.KeyVault.Commands
+{
+    internal class CamelCasePropertyNamesWithOverridesContractResolver : CamelCasePropertyNamesContractResolver
```

Comment:
These internal helper should not be put on commands folder. Please consider the way to organize whole projects.